share/spice/riffsy_emoji_search/riffsy_emoji_search.js

```diff
@@ -0,0 +1,36 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_riffsy_emoji_search = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error || !api_result.results) {
+            return Spice.failed('riffsy_emoji_search');
+        }
+        
+        Spice.add({
+            id: 'riffsy_emoji_search',
+            name: 'Riffsy Emoji Search',
+            data: api_result.results,
+            view: 'Images',
+            meta: {
+                sourceName: 'Riffsy',
+                sourceUrl: 'http://www.riffsy.com/search/' + encodeURIComponent(DDG.get_query()),
+            },
+            normalize: function(item) {
+                return {
+                    thumbnail: item.media[0].tinygif.url,
```

Comment:
Just to be on the safe side, I'd like to check for the existence of `media`, `media[0]`, and `tinygif` beforehand.